api/src/main/java/org/openmrs/util/Security.java

```diff
@@ -239,7 +239,10 @@ public static String encrypt(String text, byte[] initVector, byte[] secretKey) {
 	 * @return encrypted text
 	 * @since 1.9
 	 * <strong>Should</strong> encrypt short and long text
+	 * 
+	 * @deprecated As of version 2.3.0, this method is not referenced in openmrs-core or any other projects under github OpenMRS organisation.
```

Comment:
Since 2.3.0 was released, the deprecated version should now be 2.4.0
Could we also add that these methods are going to be removed without a replacement? That is because API clients always expect alternative recommendations after such deprecations.

Reply:
@kangbreder can you please incorporate @dkayiwa suggestions :)